index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <section>
+            <a class="logo">logos</a>
+            <nav class="humbuger"><img src="images/icones/Union.png" alt="humburger"> </nav>
+        </section>
+        <div class="contain">
+            <div class="container-title">
+                <h1 class="item1">Faustin Makutano</h1>
+                <h2 class="item2">Software Developer</h2>
+                <div class="literature">
+                    <p class="item3">I can help you build a product , feature or website Look through some of my work and experience!<br> If you like what you see and have a project you need coded,<br> don’t hestiate to contact me.</p>
+                </div>
+            </div>
+        </div>
+        <header class="border-bottom">
```

Comment:
- [ Optional ] As the generally skeleton provided in my previous comment, `<header>` is a special section type reserved for the header of the page. So, kindly use this tag at the top that contains the logo and navigation links for semantic reasons. Kindly, refer [this](https://www.w3schools.com/html/html_layout.asp) HTML5 page layout template.